docker/assets/sync_repo

```diff
@@ -15,4 +18,6 @@ BRANCH="${2:-master}"
     rm -rf "${WORKDIR}"/{.git,.github,.gitignore,LICENSE,README.md,DEAfrica_notebooks_template.ipynb} || true
     rsync --verbose --recursive "${WORKDIR}/" ~/
     rm -rf "${WORKDIR}"
+    # Install Tools folder if available
+    pip install -e REPO_DIR/Tools || true
```

Comment:
do you mean `${REPO_DIR}/Tools}`

Reply:
Also new pip complains about using script, new way is supposed to be
```
python -m pip install -e "${REPO_DIR}/Tools" || true
```

Reply:
Sure do! Will fix